### PR TITLE
apt dist-upgrade: add page

### PR DESCRIPTION
**Pull request by mandar-gite**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by spageektti** [comment]
We do not support more than 8 examples on pages. Have you taken a look at the contributing guide and the style guide?

**Comment by jxu** [comment]
You created the page `apt-upgrade.md`, which is different than `apt dist-upgrade`. This should go under `apt.md`  https://github.com/tldr-pages/tldr/blob/main/pages/linux/apt.md

**Comment by mandar-gite** [comment]
apt.md already has 8 commands .

Earlier pull request was rejected because of that .

That’s the reason i was asked make new  page .

File: pages/linux/apt.md [hunk 1]
```diff
@@ -36,3 +36,6 @@
 - List installed packages:
 
 `apt list --installed`
+
+
+
```
Comment (spageektti) [inline pages/linux/apt.md:41]:
```suggestion
```

**Review by mandar-gite — COMMENTED** [review]
deleted the new lines

**Review by mandar-gite — COMMENTED** [review]
ok

**Review by mandar-gite — COMMENTED** [review]
changed the first letter to capital

**Comment by mandar-gite** [comment]
Let me know if any other changes are required .

File: pages/linux/apt-get-dist-upgrade.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# apt-get dist-upgrade
+
+> The dist-upgrade performs comprehensive package upgrades, resolving dependency conflicts and installing/removing packages as needed.
```
Comment (spageektti) [inline pages/linux/apt-get-dist-upgrade.md:3]:
```suggestion
> Perform comprehensive package upgrades, resolving dependency conflicts and installing/removing packages as needed.
```
We avoid using the tool name in description when not needed(style guide).

**Pull request closed without merging**